src/data_retrieval/__main__.py

```diff
@@ -57,52 +58,87 @@ def get_cds_era5():
     file_name: str = (
         "era5_total_precipitation_global_1981_2023_all_months.grib"
     )
-    output_path: str = os.path.expanduser("~/Downloads/era5_global_data")
-    setup_output_path(output_path)
+    blob_path = "/raw/glb/era5/era5-total-precipitation-1981-2023.grib"
 
     # Define the years and months for the single request
-    # From 1981 to 2023 try extra 2024 next time
     years: List[int] = list(range(1981, 2024))
-
-    # All months
     months: List[int] = list(range(1, 13))
 
     # Download the data for the entire globe with a single request
-    download_era5_cds(years, months, output_path, file_name)
```

Comment:
same here. Add CLI option to stream to cloud instead of killing the local download